Apps/Admin/Server/Converters/UtcPstDateOutputConverter.cs

```diff
@@ -0,0 +1,58 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Admin.Server.Converters
+{
+    using System;
+    using System.Globalization;
+    using CsvHelper;
+    using CsvHelper.Configuration;
+    using CsvHelper.TypeConversion;
+    using HealthGateway.Common.Data.Utils;
+
+    /// <summary>
+    /// Converts UTC date to PST.
+    /// </summary>
+    public class UtcPstDateOutputConverter : ITypeConverter
```

Comment:
We should use "Pacific Time" rather than "PST" in the comments and names of types and methods, because the code should be converting to PDT or PST as appropriate for the time of the year.

(I believe the code already works this way, even though the time zone we are looking up is labelled "Pacific Standard Time" in Windows.  The [TimeZoneInfo](https://learn.microsoft.com/en-us/dotnet/api/system.timezoneinfo?view=net-7.0) that is returned by [FindSystemTimeById](https://learn.microsoft.com/en-us/dotnet/api/system.timezoneinfo.findsystemtimezonebyid?view=net-7.0) knows about whether the time zone supports daylight savings time.)

This comment may become moot if we generalize the time zone functionality by making the time zones configurable.

Reply:
Spoke with @sslaws Moved GetLocalTimeZone from JobScheduler to Common.Data.  It also uses 'Pacific Standard Time' so not changing that.